code/game/objects/effects/poster_motivational.dm

```diff
@@ -0,0 +1,291 @@
+#define POSTER_MOOD_CAT "poster_mood"
+
+/obj/item/poster/quirk
+	name = "placeholder quirk poster"
+	desc = "Uh oh! You shouldn't have this!"
+	icon_state = "rolled_poster_legit"
+	var/quirk_poster_department = NONE
+
+/obj/item/poster/quirk/Initialize(mapload, obj/structure/sign/poster/new_poster_structure)
+	. = ..()
+
+	register_context()
+
+	if(poster_structure)
+		var/obj/structure/sign/poster/quirk/department_grab = poster_structure
+		department_grab.quirk_poster_department = quirk_poster_department
+
+/// You can use any spraypaint can on a quirk poster to turn it into a contraband poster from the traitor objective
+/obj/item/poster/quirk/attackby(obj/item/postertool, mob/user, params)
+	if(!is_special_character(user))
+		return ..()
+	if(!HAS_TRAIT(user, TRAIT_POSTERBOY))
+		return ..()
+	if(!istype(postertool, /obj/item/toy/crayon))
+		return ..()
+	balloon_alert(user, "converting poster...")
+	if(!do_after(user, 5 SECONDS, user))
+		balloon_alert(user, "interrupted!")
+		return
+	var/obj/item/poster/traitor/quirkspawn = new(get_turf(src))
+	user.put_in_hands(quirkspawn)
+	to_chat(user, span_notice("You have converted one of your posters!"))
+	qdel(src)
+
+/// Screentip for the above
+
+/obj/item/poster/quirk/add_context(atom/source, list/context, obj/item/held_item, mob/user)
+	if(!is_special_character(user))
+		return NONE
+	if(!HAS_TRAIT(user, TRAIT_POSTERBOY))
+		return NONE
+	if(!istype(held_item, /obj/item/toy/crayon))
+		return NONE
+	context[SCREENTIP_CONTEXT_LMB] = "Turn into Demoralizing Poster"
+	return CONTEXTUAL_SCREENTIP_SET
+
+/obj/structure/sign/poster/quirk
+	name = "quirk poster"
+	desc = "A large piece of homemade space-resistant printed paper."
+	poster_item_name = "homemade poster"
+	poster_item_desc = "A poster made with love, some people will enjoy seeing it."
+	poster_item_icon_state = "rolled_legit"
+	var/quirk_poster_department = NONE
+	var/datum/proximity_monitor/advanced/quirk_posters/mood_buff
+
+/obj/structure/sign/poster/quirk/on_placed_poster(mob/user)
+	mood_buff = new(src, 7, TRUE, quirk_poster_department)
+	return ..()
+
+/obj/structure/sign/poster/quirk/attackby(obj/item/I, mob/user, params)
+	if (I.tool_behaviour == TOOL_WIRECUTTER)
+		QDEL_NULL(mood_buff)
+	return ..()
+
+/obj/structure/sign/poster/quirk/Destroy()
+	QDEL_NULL(mood_buff)
+	return ..()
+
+/obj/structure/sign/poster/quirk/apply_holiday()
+	var/obj/structure/sign/poster/traitor/holi_data = /obj/structure/sign/poster/quirk/festive
+	name = initial(holi_data.name)
+	desc = initial(holi_data.desc)
+	icon_state = initial(holi_data.icon_state)
```

Comment:
```suggestion
/obj/structure/sign/poster/quirk/apply_holiday()
	var/obj/structure/sign/poster/traitor/holiday_data = /obj/structure/sign/poster/quirk/festive
	name = initial(holiday_data.name)
	desc = initial(holiday_data.desc)
	icon_state = initial(holiday_data.icon_state)
```

no need to abbreviate, four letters is nothing.

Reply:
btw the other holiday poster stuff (in demotivational posters and the regular poster files) use this format bc thats what the original holiday poster pr set it to

Reply:
Ah, I see. We just wanna stray away from the pattern in newly added code, we can fix the old weird stuff later